src/gpodder/gtkui/base.py

```diff
@@ -114,7 +114,7 @@ def main(self):
         """
         Gtk.main()
 
-    def quit(self):
+    def quit_app(self):
```

Comment:
why this change?

Reply:
I couldn't find any actual use of `main()` or `quit()` on a cursory search.
Do you know if they are called?

Reply:
I also could not find a user for this function. The reason for the change is: Fixes flake8 'A003 class attribute "quit" is shadowing a Python builtin'.